### PR TITLE
 ui: clickable ElidedLabel

### PR DESCRIPTION
**Pull request by deanlee**
we can overriding  mousePressEvent if need press state in this control.

### CODE REVIEW

**Comment by sshane** [comment]
Much cleaner, I'll check it out soon!

**Review by sshane — APPROVED** [review]

File: selfdrive/ui/qt/offroad/networking.cc [hunk 1]
```diff
@@ -247,12 +247,12 @@ void WifiUI::refresh() {
     hlayout->setSpacing(50);
 
     // Clickable SSID label
-    QPushButton *ssidLabel = new QPushButton(network.ssid);
```
Comment (adeebshihadeh) [inline selfdrive/ui/qt/offroad/networking.cc:250]: can't we just make this a label?

**Comment by adeebshihadeh** [comment]
any update on this?

**Comment by deanlee** [comment]
done

File: selfdrive/ui/qt/widgets/controls.h [hunk 2]
```diff
@@ -23,6 +23,22 @@ class ElidedLabel : public QLabel {
...
 
+
+class ClickableLabel : public ElidedLabel {
+  Q_OBJECT
+
+public:
+  explicit ClickableLabel(const QString &text, QWidget *parent = 0) : ElidedLabel(text, parent) {}
+
+signals:
+  void clicked();
```
Comment (adeebshihadeh) [inline selfdrive/ui/qt/widgets/controls.h:34]: can't we just add a clicked signal to ElidedLabel?

**Pull request merged**